integration_tests/rest_api_utils.py

```diff
@@ -176,7 +176,7 @@ def my_model(**keyword_args):
     "cpus": 20,
     "gpus": 1,
     "memory": "20Gi",
-    "gpu_type": "nvidia-ampere-a10",
+    "gpu_type": "nvidia-hopper-h100-1g20gb",
```

Comment:
this might fail on training cluster?

Reply:
i'm going to update the justfile to run on prod cluster